lib/core/logger.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/foundation.dart';
+import 'package:logger/logger.dart';
+
+final logger = Logger(
+  level: kDebugMode ? Level.trace : Level.off,
```

Comment:
Ce qui veut dire qu'une fois en prod on ne pourrait jamais demander de traces aux utilisateurs en cas de crash, non ?

Reply:
Et tant qu'à désactiver les logs en prod je mettrais cette variable nullable, et à appeler avec `logger?` du coup

Reply:
TODO : add a level which allows :
- in dev mode : all levels
- in production mode : only levels warning, error, fatal error